pkg/apis/batch/validation/validation.go

```diff
@@ -281,7 +284,7 @@ func ValidateJobStatusUpdate(status, oldStatus batch.JobStatus) field.ErrorList
 func ValidateCronJob(cronJob *batch.CronJob, opts apivalidation.PodValidationOptions) field.ErrorList {
```

Comment:
rename this to ValidateCronJobCreate to make it clear it should *only* be called when validating a new CronJob, not during an update API request (will need to update the single non-test caller in cronJobStrategy#Validate and unit test callers)